index.js

```diff
@@ -65,9 +65,9 @@ client.on('message', async msg => {
 
 	// Prevent commands from being run after hours
 	let now = new Date();
-	if (now.getHours() < 9 || now.getHours() > 17 || now.getDay() == 0 || now.getDay() == 6) {
+	if (now.getHours() < 9 || now.getHours() > 17 || now.getDay() != 5) {
 		if (msg.content.toLowerCase().startsWith("!") && !mentor) {
-			msg.reply("Mentoring is closed for the day. Please check back between 10 am and 6 pm M-F")
+			msg.reply("Sorry but we currently only offer online mentoring on Fridays 10-6. However Monday through Thursday we offer mentoring in the SSE 10-6. If you have just a quick question feel free to post it in the general questions channel and someone might be able to help!")
```

Comment:
This line goes against style standards set for the repo

Reply:
To go a bit further:
- Days of the week are shown with letters, not full name
- am and pm follow any times that are mentioned

I would personally also add in a link to the general questions channel `<#ChannelId>` instead of writing out the channel name.

Reply:
To further add, this was merged without review (aka a force merge) which is against this sub-repos policy of requiring review before being merged. For this and for all the reasons above this PR should be rolled back